### ISSUE TITLE
Do I need to set "shuffle: false" in the data layer in both train_test.prototxt and test.prototxt?

### ISSUE THREAD

**Issue by icbcbicc** [chuckcho/video-caffe#29]
If I set "shuffle: true", will the order of the 16 frames be disrupted?
Thanks.

**Comment by chuckcho** [comment]
no. shuffle works on a `clip` base

**Comment by icbcbicc** [comment]
Thank you so much.
By the way, it seems that the training log has 2 kinds of Testing net(Testing net #0 and Testing net #1). 
But In the original caffe, there's only one Testing net. 
Does Testing net #0 mean testing on the training set while Testing net #1 stands for testing on the validation set? 
I guess so because the accuracy of Testing net #0 is always higher than the one of Testing net #1 .

Thanks again.

**Comment by chuckcho** [comment]
@icbcbicc that's correct. Testing net#0 is for testing-on-training-set, and net#1 is for testing-on-testing-set. Just to get a sense of how much over- or under-fit is going on.
